conf/cassandra.yaml

```diff
@@ -286,7 +286,11 @@ counter_cache_save_period: 7200
 # If not set, the default directory is $CASSANDRA_HOME/data/saved_caches.
 # saved_caches_directory: /var/lib/cassandra/saved_caches
 
-# commitlog_sync may be either "periodic" or "batch." 
+# Max amount of time in seconds server has wait for cache to load while starting the Cassandra process.
+# Cache will not load (during startup) if this value is negative as there is no time left to load.
+# max_cache_load_time: 30
```

Comment:
maybe name this `max_cache_load_time_seconds` to make it clearer

Reply:
I'm :+1: on the suggested name, or just call it a timeout `cache_load_timeout_seconds`?